R/pkg/R/context.R

```diff
@@ -345,7 +351,7 @@ spark.getSparkFilesRootDirectory <- function() {
 #'}
 #' @note spark.getSparkFiles since 2.1.0
 spark.getSparkFiles <- function(fileName) {
-  callJStatic("org.apache.spark.SparkFiles", "get", as.character(fileName))
+  file.path(spark.getSparkFilesRootDirectory(), as.character(fileName))
```

Comment:
I changed to call ```SparkFiles``` if running on driver like before, here we do the best possible to use the Scala implementation. With respect to running on executors, we have no better choice.
We already have test coverage for ```spark.getSparkFiles``` on driver and executors. To some illegal arguments, such as fileName is absolute path, it can't handle in Scala API as well.